scripts/flatpak/org.libelektra.kdb.yaml

```diff
@@ -0,0 +1,50 @@
+app-id: org.libelektra.kdb
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: /app/bin/kdb
+finish-args:
+  - --socket=wayland
+  - --socket=x11
+  - --filesystem=host
+  - --env=LUA_CPATH="/app/lib/lua/5.2/?.so;"
+build-options:
+  build-args:
+    - --share=network
+modules:
+  - name: discount
+    sources:
+      - type: archive
+        url: http://www.pell.portland.or.us/~orc/Code/discount/discount-2.2.7.tar.bz2
+        sha256: b1262be5d7b04f3c4e2cee3a0937369b12786af18f65f599f334eefbc0ee9508
+    buildsystem: simple
+    cleanup:
+      - '*'
+    build-commands:
+      - ./configure.sh --prefix .
+      - make
+      - make install
+      - mkdir -p /app/lib /app/bin /app/include
+      - cp -a bin/* /app/bin
```

Comment:
This looks hackish, isn't it possible to set the prefix to /app? (If it is not possible, a small comment why you do it this way wouldn't harm)

Reply:
It was possible to set the prefix and remove the hacky solution.